cioppino/twothumbs/browser/templates/login_to_rate.pt

```diff
@@ -1,10 +1,3 @@
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
-      xmlns:tal="http://xml.zope.org/namespaces/tal"
-      xmlns:metal="http://xml.zope.org/namespaces/metal"
-      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-      lang="en"
-      i18n:domain="cioppino.twothumbs">
```

Comment:
Sorry for the delay. Was busy with some other tasks. I got your message and added the html tags back with a tal:omit-tag expression. Hope that's fine.